bin/functions.sh

```diff
@@ -29,6 +29,11 @@ function stop() {
 }
 
 
-function goIntoContainer() {
-    docker exec -t -i ${APP_CONTAINER} bash;
+function goIntoContainerCurrentUser() {
+    docker exec -t -i -u `id -u $UID` ${APP_CONTAINER} bash;
 }
+
+
+function goIntoContainerDefaultUser() {
+    docker exec -t -i ${APP_CONTAINER} bash;
+}
```

Comment:
Missing new line on end of file. When you add next changes then will be deleted (from view of Git) line with `}`.